### ISSUE TITLE
Investigate the possible usage of framework with phar archives

### ISSUE THREAD

**Issue by lisachenko** [goaop/framework#117]
@matthiasnoback suggests an interesting idea to use phar archives with woven aspects in https://github.com/cordoval/gush/issues/17. This approach can be very promising for standalone phar applications, so need to investigate it.

**Comment by cordoval** [comment]
:+1:

**Comment by lisachenko** [comment]
So, I can confirm that framework can normally work with phar archives after implementing some dirty fixes for phar (https://bugs.php.net/bug.php?id=52769).
There is also one more bug in PHP, related to the custom stream filters that are applied to a phar stream, I will report it, when extract it. I have also implemented test symfony/console application with AOP and mark some methods with an annotation "@DryRunMethod" and apply an aspect that prevent execution of this methods if --dry-run key was given in the command line.
After that I packed source code into phar and execute it, seems good :)
